src/game/puppyprint.h

```diff
@@ -119,6 +119,7 @@ enum PuppyFont {
     FONT_NUM
 };
 
+extern u8 sDebugMenu;
```

Comment:
Letting this one slide because the below variables aren't ifdef guarded properly either, but it does bother me >:(